borsh/Cargo.toml

```diff
@@ -36,8 +36,8 @@ borsh-derive = { path = "../borsh-derive", version = "~1.5.4", optional = true }
 
 # hashbrown can be used in no-std context.
 # NOTE: There is no reason to restrict use of older versions, but we don't want to get
-# sudden breaking changes with an open range of versions, so we limit the range by not yet released 0.15.0 version:
-hashbrown = { version = ">=0.11,<0.15.0", optional = true }
+# sudden breaking changes with an open range of versions, so we limit the range by not yet released 0.16.0 version:
+hashbrown = { version = ">=0.11,<0.16.0", optional = true }
```

Comment:
Oh well, this is a range requirement, so nothing can go wrong with borsh limiting hashbrown's version, 
so i'll just double check the tests pass with latest hashbrown, then merge it